comp/networkpath/npcollector/npcollectorimpl/npcollector_test.go

```diff
@@ -457,7 +457,7 @@ func Test_npCollectorImpl_ScheduleConns(t *testing.T) {
 				},
 			},
 			expectedPathtests: []*common.Pathtest{
-				{Hostname: "10.0.0.4", Port: uint16(80), Protocol: "tcp", SourceContainerID: "testId3"},
+				{Hostname: "10.0.0.4", Port: uint16(80), Protocol: "TCP", SourceContainerID: "testId3"},
```

Comment:
Should this and the above test be `payload.TCP`?

Reply:
indeed, fixed